index.html

```diff
@@ -0,0 +1,140 @@
+    <title>Music concert Home Page</title>
+    <link rel="stylesheet" type="text/css" href="index.css">
+    <link rel="stylesheet" href="node_modules/bootstrap/dist/css/bootstrap.min.css">
+    <script src="https://kit.fontawesome.com/0f56b98a6f.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <header>
+        <div class="header1">
+            <i class="fa-brands fa-facebook-f"></i>
+            <i class="fa-brands fa-twitter"></i>
+            <a href="#">English</a>
+            <a href="#">My Page</a>
+            <a href="#" class="logout">Logout</a>
+        </div>
+        <div class="header2">
+        <div class="header-logo">
+            <img src="Images/white logo.png">
+        </div>
+        <div>
+            <button class="hambuger">
+                <i class="fa-solid fa-bars"></i>
+            </button>
+        </div>
+        <nav class="nav-menu">
+            <button class="close">
+                <i class="fa-solid fa-xmark"></i>
+            </button>
+            <ul>
+                    <li><a href="About.html" class="item-about">About</a></li>
+                    <li><a href="#" class="item-program">Program</a></li>
+                    <li><a href="#" class="item-join">Join</a></li>
+                    <li><a href="#" class="item-sponsor">Sponsor</a></li>
+                    <li><a href="#" class="item-newa">News</a></li>
+                   
+                </ul>
+                </nav>
+            <button class="campaign">
+                Referral program
+            </button>
+        </div>
+    </header>
+    <section class="home-about">
+        <div class="home2">
+        <h3>"Join the Musical Celebration, Experience Unity!"</h3>
+        <h1>HARMONY IN MOTION 2023</h1>
+        <p class="description">Get ready for the musical event of the year, "Harmony in Motion 2023"! This musical extravaganza will bring together the beats of various genres to create a symphony of sound. Join us on the evening of August 20, 2023 , for an unforgettable night of unity and rhythm. As we come together to celebrate the power of music, let's create memories that will last a lifetime</p>
+        <p class="date">
+            2023.8.20(SAT) ~ 21(MON)<br><span>
+            @Grand Park Concert Hall, downtown Los Angeles
+        </span></p>
+    </div>
+    </section>
+    <section class="Main-Program">
+        <div class="title-line">
+        <h1>Main Program</h1>
+        <hr>
+    </div>
+        <div class="Row">
+            <div class="col-12 custom-col-12">
+                <div class="content">
+                    <img src="Images/icons8-spotify-200.png" alt="spotify-icon">
+                    <h2>Local Musicians</h2>
+                    <p>The opening act will feature talented up-and-coming musicians from the local area, kicking off the night with their lively and energetic performances.</p>
+                </div>
+            </div>
```

Comment:
- [ ] Good job working on the main program section, but there are still a few things to take care of. They are listed below.
- [ ] At the moment, the sizes of the program cards does not match the original design
- [ ] Please kindly help give the cards proper padding to give the text enough breathing space.
- [ ] Please kindly ensure that the spacing between each card matches the original design.
- [ ] Inside the cards, please kindly ensure the image and text items are centered so that it matches the original design.
- [ ] Also, the button design on the large screens should match that of the original design

### Your Design
![image](https://user-images.githubusercontent.com/43995830/219870733-dfac3783-dd27-4663-b355-dd660b9a509d.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/219870780-7d2f1101-be66-48e3-a3fb-acbbf4dfe2a4.png)